Week1/homework/databaseClasses.js

```diff
@@ -0,0 +1,118 @@
+'use-strict';
+
+const con = require('./connectDatabase').connection;
+const pool = require('./connectDatabase').pool;
+const countries = require('./country.json').country;
+const cities = require('./city.json').city;
+
+class Database {
+  async createDatabase(db) {
+    try {
+      await con.query(`CREATE DATABASE IF NOT EXISTS ${db}`);
+      console.log('world database is created and selected successfully');
+    } catch (e) {
+      console.log('world database could not create', e.stack);
+    } finally {
+      con.end();
+    }
+  }
+}
+
+class CountrySQL {
+  async createCountriesTable() {
+    try {
+      const q = `CREATE TABLE IF NOT EXISTS countries (
+      Code VARCHAR(3) PRIMARY KEY,
+      Name VARCHAR(60),
+      Continent VARCHAR(60),
+      Region VARCHAR(60),
+      SurfaceArea FLOAT,
+      IndepYear INT,
+      Population INT,
+      LifeExpectancy FLOAT,
+      GNP FLOAT,
+      GNPOld FLOAT,
+      LocalName VARCHAR(60) ,
+      GovernmentForm VARCHAR(60),
+      HeadOfState VARCHAR(80),
+      Capital varchar(80),
+      Code2 VARCHAR(2)
+    ) DEFAULT CHARSET = utf8mb4;`;
+
```

Comment:
I guess you did put this encoding on purpose. I didn't know about it but apparently you are right, it is recommended to encode correctly.

Reply:
Your create statement is really good.
I know that in this first week lesson you have seen teacherS table but it's more common to use singular for table name, like here city and country. But it's just a convention.

Reply:
If i didnt put charset it would throw errors in some columns.